no-computer.md

```diff
@@ -58,12 +109,33 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This language is unique in that almost the entirety of the design process would
+be consumed by creating the vocabulary, the "keywords". In this case,
+"keywords" are short stories of several hundred words or more. While designing
+the keywords is technically a part of designing the language, it fails to
+address many of the interesting parts of language design which we're learning
+about in this class.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This depends entirely on how big the "starter set" of stories is going to be.
+It would not be difficult to write, say, 10 to 20 short stories in the course
+of the semester. It would be impossible to write 50, especially since the
+difficulty of the task will increase exponentially with the number of stories
+as coherency and consistency becomes an issue.
```

Comment:
I don't think it is too ambitious. I don't think you should even think about writing more than 10 to 20 example stories. I would certainly be interested in using your DSL!